discovery-provider/es-indexer/src/indexers/TrackIndexer.ts

```diff
@@ -121,11 +129,12 @@ export class TrackIndexer extends BaseIndexer<TrackDoc> {
       ) as saved_by
     
     from tracks
-    join users on owner_id = user_id 
-    join aggregate_user on users.user_id = aggregate_user.user_id
-    join aggregate_plays on tracks.track_id = aggregate_plays.play_item_id
-      WHERE tracks.is_current = true 
-        AND users.is_current = true
+      join users on owner_id = user_id 
+      left join aggregate_user on users.user_id = aggregate_user.user_id
```

Comment:
same here

Reply:
just to get `track.artist. follower_count`